keys/README.md

```diff
@@ -0,0 +1 @@
+This folder contains the identity files.
```

Comment:
Can't/shouldn't we have spark-ec2 search in the default `~/.ssh/`, or in a user-specifiable directory?